eslint-plugin/rules/component-properties/index.js

```diff
@@ -28,6 +38,28 @@ module.exports = {
       })
     }
 
+    function reportSetterProps(node) {
+      const { name } = node.name
+
+      if (isSetter(name)) {
+        context.report({
+          message: messages['no setters'](name),
+          node,
+        })
+      }
+    }
+
+    function reportDestructuredSetterProps(node) {
+      node.argument.properties.forEach(x => {
```

Comment:
Dit gaat mis wanneer het deze structuur is: `{...someProps}`. Even een test case toevoegen.